package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "next-compression-plugin",
```

Comment:
To keep consistency with other packages and after the discussion about scoped packages we should call this `@moxy/next-compression`.